jobs/microk8s/clusterapi/microk8s.profile

```diff
@@ -0,0 +1,29 @@
+name: microk8s
```

Comment:
Why do we have a copy of this file here? Should we get it from the main repository so we do not need to maintain it in two places?